app/src/main/java/co/nimblehq/compose/crypto/ui/components/linechart/CoinPriceChart.kt

```diff
@@ -0,0 +1,143 @@
+                    yAxisDrawableArea = yAxisDrawableArea,
+                    size = size,
+                    offset = horizontalOffset
+                ).copy(left = 0F) // Chart should fill the screen width
+
+                lineDrawer.drawLine(
+                    drawScope = this,
+                    canvas = canvas,
+                    linePath = computeLinePath(
+                        drawableArea = chartDrawableArea,
+                        lineChartData = lineChartData,
+                        transitionProgress = transitionAnimation.value
+                    )
+                )
+                lineShader.fillLine(
+                    drawScope = this,
+                    canvas = canvas,
+                    fillPath = computeFillPath(
+                        drawableArea = chartDrawableArea,
+                        lineChartData = lineChartData,
+                        transitionProgress = transitionAnimation.value
+                    )
+                )
+
+                val maxPrice = lineChartData.points.maxOf { it.value }
+                val minPrice = lineChartData.points.minOf { it.value }
+                val maxPriceIndex = lineChartData.points.indexOfFirst { it.value == maxPrice }
+                val minPriceIndex = lineChartData.points.indexOfFirst { it.value == minPrice }
+
+                lineChartData.points.forEachIndexed { index, point ->
+                    withProgress(
+                        index = index,
+                        lineChartData = lineChartData,
+                        transitionProgress = transitionAnimation.value
+                    ) {
+                        val pointLocation = computePointLocation(
+                            drawableArea = chartDrawableArea,
+                            lineChartData = lineChartData,
+                            point = point,
+                            index = index
+                        )
+                        pointDrawer.drawPoint(
+                            drawScope = this,
+                            canvas = canvas,
+                            center = pointLocation
+                        )
+                        if (index in listOf(minPriceIndex, maxPriceIndex)) {
+                            labelDrawer.drawLabel(
+                                drawScope = this,
+                                canvas = canvas,
+                                label = point.label,
+                                pointLocation = pointLocation,
+                                xAxisArea = xAxisDrawableArea,
+                                isHighestPrice = index == maxPriceIndex
+                            )
+                        }
+                    }
+                }
+            }
+        }
+
+        // Chart intervals
+        ChartIntervalsButtonGroup(
+            modifier = Modifier
+                .padding(top = 24.dp),
+            onIntervalChanged = onTimeIntervalChanged
+        )
```

Comment:
I feel like this chart interval buttons group is not a part of chart customization. How about moving it out of this custom chart and putting it in DetailScreen?

![image](https://user-images.githubusercontent.com/16315358/195397377-d791b905-6379-4504-9fcf-8a5ebcda3126.png)

Reply:
@luongvo Yeah. Initially, I added the group of Interval buttons inside the Details screen then I decided to move it into the chart. 🤔 

Let me see...We could change the chart library so better to move the group outside. That's make sense.